ludwig/contribs/wandb.py

```diff
@@ -40,10 +40,10 @@ def import_call(*args, **kwargs):
                 "Ignored --wandb: Please install wandb; see https://docs.wandb.com")
             return None
 
-    def train_model(self, model, *args, **kwargs):
+    def train_model(self, model, model_definition, *args, **kwargs):
```

Comment:
model here is a NewLudwigModel instance right? then instead of having another parameter we can obtain the model definition with `model.model_definition` I believe

Reply:
No, it's `ECD`.

Reply:
got it, then let's keep it the way it is for now. Maybe later we can consider switching model to an LudwigModel if it makes sense.